include/linker/ramfunc.ld

```diff
@@ -0,0 +1,17 @@
+SECTION_DATA_PROLOGUE(.ramfunc,,)
+{
+#ifdef CONFIG_CPU_HAS_MPU
```

Comment:
As I said, this is not the right Kconfig option.
You could use #if CPU_HAS_ARM_MPU (which is a user option to build with MPU support)

Otherwise, you could just do nothing, and simply align with MPU_GUARD_ALIGN_AND_SIZE in line 10)

Reply:
Actually, I would prefer the second option, i.e. no #ifdef CPU_HAS_ARM_MPU, but only the alignments, depending on MPU_GUARD_ALIGN_AND_SIZE and MPU_REQUIRE_SPOWER_OF_TWO_ALIGNMENT